web/routes/sign_in.rb

```diff
@@ -6,7 +6,9 @@ class DryTwitter::Web
 
     r.post do
       r.resolve "sign_in.sign_in" do |sign_in|
-        sign_in.call(r.params) do |m|
+        params_and_env = r.params.merge({})
```

Comment:
This should go in a operation, no logic in your route. In the end we want this to be a single line.

Reply:
I have to somehow pass env into the operation. It is available on Roda only.

Reply:
I the meantime I refactored to use to session instead. But I still have to have this chunk of code in order to pass the session to my operation. 

Alternatively I could drop the specialized cookie operation and do as in Berg, and set the cookie on the route. Their solution can also be debated is not clear enough since the routing have multiple responsibilities. I choose to keep mine solution because it was the first one I implemented.